webdriver-spec.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+﻿<!doctype html>
```

Comment:
I can’t tell from the raw patch what has actually changed here.

Reply:
It’s probably the UTF-8 byte marker from the editor I was using, since it’s at the very beginning of the file. I can fix that up if need be, I suppose.

Reply:
Yeah, we don’t want a BOM there.

Reply:
Removed in subsequent commit.